src/styles/modifiers.css

```diff
@@ -0,0 +1,19 @@
+.stars--1 .stars__star:nth-child(1) {
```

Comment:
Remove the space before `:nth-child(1)` to ensure it applies directly to `.stars__star` as a child of `.stars--1`.